### DISCUSSION TITLE
Why is my device diagnostics showing "Away"?

### DISCUSSION THREAD

**Discussion by odhiambo** [mKeRix/room-assistant#1095] [General]
Hi good people,
I have set up RA on top of my HA running on RPi4.
I only want to track the presence of my phone at the moment. My config is:

global:
  instanceName: Office Pi
  integrations:
    - homeAssistant
    - bluetoothLowEnergy
homeAssistant:
  mqttUrl: 'mqtt://10.10.12.103:1883'
  mqttOptions:
    username: mqtt
    password: XXXXXX
bluetoothLowEnergy:
  allowlist:
    - a4e6ab7c3e964ccfa18ea025xxxxxxxx-100-1
  maxDistance: 5
  tagOverrides:
    a4e6ab7c3e964ccfa18ea025xxxxxxxx-100-1:
      name: RedmiNote9Pro

I can see the devices detected from the RA logs and this device is one of them. However, when I check the device details, the diagnostic part shows the "Away", which I don't understand. TBH I haven't read through the whole of RA manual, but what does the "Away" mean? The device is right next to my RPi4 - on the same table.

Thanks in advance.

**Comment by mKeRix** [comment]
Based on the config you provided I would guess that the device is marked as away because the reported distance is higher than what you configured for `maxDistance`. You can look at the [API](https://www.room-assistant.io/guide/api.html) to get detailed information about the readings.

The distance is calculated based on a base value which should represent the signal strength at 1m. This value is probably not sent by your phone, so a default is being used, which may not match reality and lead to wrong distance calculations. If my memory serves correctly the HA Android App had a power setting for its beacon feature, increasing that might also help.

**Reply by odhiambo** [reply]
I thought that the maxDistance is a value in meters, so 5 is 5 meters. My RPi4 and my phone are barely 1m apart. They are together on my work table.

**Reply by mKeRix** [reply]
It is indeed in meters, but the distance value is calculated by room-assistant with a base value as mentioned above. If that base value doesn't match the reality (it should be set to the signal strength when the device is 1 meter away) the calculations will be off.

An alternative way of testing this is removing the `maxDistance` setting from your config temporarily and then looking at the attributes of your entity in Home Assistant.

**Reply by odhiambo** [reply]
When the phone is about 5m away from the RPi, I see the RSSI as -55. When I bring it very close to the RPi, the RSSI is around -46. So ideally, I want to tell RA that if the RSSI is over -55 and above, then I am NOT in this room.  If it's around -49 or -46, then I am in this room. I live in a small rented apartment.
How do I tell RA this?
Can I use the RSSI and ignore/do away with the maxDistance value?

Thanks for spending your time on this.

**Reply by odhiambo** [reply]
I am happy to report that I got this to work by setting:
maxDistance: 6
minDiscoveryLogRssi: -55

TBH, I am not sure which of the two is making it work :-)

**Reply by mKeRix** [reply]
Happy to hear you got it working! For future reference, you may be able to get better distance predictions by configuring the measured signal strength at 1m away as you mentioned:

```yaml
bluetoothLowEnergy:
  tagOverrides:
    a4e6ab7c3e964ccfa18ea025xxxxxxxx-100-1:
      name: RedmiNote9Pro
      measuredPower: -55
```

No need to mess with a running system though. :)
